src/smexperiments/tracker.py

```diff
@@ -167,6 +167,7 @@ def load(
     @classmethod
     def create(
         cls,
+        trial_component_name="TrialComponent",
```

Comment:
you should take another look at the current behavior: https://github.com/aws/sagemaker-experiments/blob/main/src/smexperiments/_utils.py#L44-L52

Reply:
Gotcha. I can rename the parameter to trial_component_name_prefix. I can see how just trial_component_name would be misleading since the name is appended with a timestamp and random number. 

I could also go the route of adding conditional logic that skips appending the timestamp and random numbers if the user specifies something other than "TrialComponent". That'll leave it up to the user to add their own timestamp if they wish.

Do you have a preference for either option?